ui/core/public/user.html

```diff
@@ -1,20 +1,25 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel=stylesheet href="https://fonts.googleapis.com/icon?family=Material+Icons">
-    <link href="https://fonts.googleapis.com/css?family=Work+Sans:300,400,500,700" rel=stylesheet>
-    <link href=https://use.fontawesome.com/releases/v5.0.8/css/all.css rel=stylesheet>
-    <link rel="icon" href="/static/favicon.ico">
-    <title>proxeus</title>
-  </head>
-  <body>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width,initial-scale=1.0">
+  <link rel=stylesheet href="https://fonts.googleapis.com/icon?family=Material+Icons">
+  <link href="/api/appearance-css" rel=stylesheet id="customstylelink">
```

Comment:
There is a performance hit to hitting this service link every page load. Please put the <link> at the end of the body to at least benefit from browser loading priority. If it becomes a problem, I would suggest writing to a statically loaded file (like the favicon) on app startup.

Reply:
but it should be there because it should override default styles (should be loaded at the first place along with the main js scripts)

Reply:
it's not compiling every time you request that endpoint. you can thinks like it's a static file

Reply:
fixed now